apps/provider-console/src/components/dashboard/FinanceCard.tsx

```diff
@@ -0,0 +1,61 @@
+import { Card, CardContent, Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from "@akashnetwork/ui/components";
+
+interface PercentChangeProps {
+  currentPrice: number | null;
+  previousPrice: number | null;
+}
+
+interface FinanceCardProps {
+  title: string;
+  subtitle: string;
+  currentPrice: number | null;
+  previousPrice: number | null;
+  message: string | null;
+}
+
+const PercentChange: React.FC<PercentChangeProps> = ({ currentPrice, previousPrice }) => {
+  if (currentPrice === null || previousPrice === null || previousPrice === 0) {
+    return <span className="text-gray-500">0%</span>;
+  }
+
+  const percentageChange = ((currentPrice - previousPrice) / previousPrice) * 100;
+  const formattedChange = Math.abs(percentageChange).toFixed(2);
```

Comment:
feels like some `useMemo` would be useful

Reply:
Done